src/static/routes-to-urls/__tests__/index.js

```diff
@@ -0,0 +1,104 @@
+// @ flow
```

Comment:
Yeah probably a mistake. No big deal, no tests are "flowed" yet. Will revisit when refactoring for monorepo.